requirements.txt

```diff
@@ -2,18 +2,19 @@ alembic>=1.3.1, <2.0.0
 arrow>=0.14.6, <2.0.0
 defusedxml>=0.6.0, <1.0.0
 fedora_messaging>=2.0.1, <3.0.0
-flask>=1.1.4, <3.0.0
 flask-login>=0.4.1, <1.0.0
 flask-wtf>=0.14.2, <1.0.0
+flask>=1.1.4, <3.0.0
 jinja2>=2.11.2, <4.0.0
 ordered-set>=3.1, <5.0.0
 python-dateutil>=2.8.0, <3.0.0
-pytoml>=0.1.18, <1.0.0
 semver>=2.8.1, <3.0.0
-social-auth-app-flask>=1.0.0, <2.0.0
 social-auth-app-flask-sqlalchemy>=1.0.1, <2.0.0
+social-auth-app-flask>=1.0.0, <2.0.0
+sqlalchemy>=1.3.17, <2.0.0
```

Comment:
Why the changes in ordering?

Reply:
I ran `:sort` to sort stuff alphabetically, so it is quicker to find the keywords. When I add a new item to requirements, I usually write it in the top and then run `sort`.

Reply:
But `flask` should be placed before `flask-login`, so that `:sort:` is doing wrong.

Reply:
@abitrolly Yes, I would assume the same. that `flask` should before `flask-login`

Reply:
I think in case there would be whitespaces after the name, the sort would work correctly.

Reply:
Should I add whitespaces so the sort works or reorder it manually and keep it without spaces?

Reply:
I never used `sort` on requirements file, but it sounds like a good idea. I wonder what will happen if we run `black` on requirements.txt

Reply:
It can't be parsed by `black` unfortunately

Reply:
@LenkaSeg Let's use the manual ordering for this PR and if you want, you can create a separate PR for changing the requirements file to be sortable by `sort`

Reply:
Ok, returning it back.